coqpit/coqpit.py

```diff
@@ -432,6 +437,11 @@ def deserialize_immutable(cls, data: dict) -> "Serializable":
                 if field.name in vars(cls):
```

Comment:
This check does not get the complex type (like List[int] and others). The default values of complex type fields will be ignored. To deal with that, I added here a check if the default value is not "MISSING" we use it instead of raising the error. So if we need that the user provides some param it needs to be defined as "MISSING" string or object.